common/seo.js

```diff
@@ -1,72 +1,60 @@
     createDescription: () =>
-      `Terms of Service of SikhiToTheMax. ${basicDescription}`,
+      `Terms of Service of SikhiToTheMax. ${BASIC_META_DESCRIPTION}`,
   },
   '/about': {
-    title: suffixAppName('About'),
+    createTitle: () => suffixAppName('About'),
     createDescription: () =>
-      `SikhiToTheMax is backed by a non-profit organization named Khalis Foundation. ${basicDescription}`,
+      `SikhiToTheMax is backed by a non-profit organization named Khalis Foundation. ${BASIC_META_DESCRIPTION}`,
   },
   '/ang': {
-    title: req => suffixAppName(`Ang ${getAng(req)} of ${getSource(req)}`),
-    createDescription: req =>
-      `Read page number ${getAng(req)} of ${getSource(
-        req
-      )} now. ${basicDescription}`,
+    createTitle: (title = 'Ang') => suffixAppName(title),
+    createDescription: (description = 'Read page number ') => `${description}. ${BASIC_META_DESCRIPTION}`,
   },
   '/index': {
-    title: suffixAppName('Index'),
+    createTitle: () => suffixAppName('Index'),
     createDescription: () =>
       `Index page of Sri Guru Granth Sahib Jee, Sri Dasam Granth. Read Asa Ki Vaar, Raag Dhanasari, Ramkali Ki Vaar Rai Balvand, Shalok Sehskritee, Shalok Vaaran Te Vadeek, Salok Mahalla 9, Raag Maala, Akaal Ustat, Vaar Sri Bhagouti Jee kee, Sri Shastar Naam Mala, Zafarnamah, Khalsa Mahima, Sri Charitropakhyan`,
   },
   '/help': {
-    title: suffixAppName('Help'),
+    createTitle: () => suffixAppName('Help'),
     createDescription: () =>
-      `Need help in understanding how to use SikhiTheMax? ${basicDescription}`,
+      `Need help in understanding how to use SikhiTheMax? ${BASIC_META_DESCRIPTION}`,
   },
   '/hukamnama': {
-    title: suffixAppName('Hukamanama'),
+    createTitle: () => suffixAppName('Hukamanama'),
     createDescription: () =>
-      `Read Sri Mukhwaakh Hukamnama of today from Sri Darbar Sahib, Harmandir Sahib (Golden Temple) Amritsar online now. ${basicDescription}`,
+      `Read Sri Mukhwaakh Hukamnama of today from Sri Darbar Sahib, Harmandir Sahib (Golden Temple) Amritsar online now. ${BASIC_META_DESCRIPTION}`,
   },
   '/search': {
-    title: suffixAppName('Search Results'),
+    createTitle: () => suffixAppName('Search Results'),
     createDescription: () =>
       `Search Sri Guru Granth Sahib Jee, Sri Dasam Granth Sahib, Bhai Gurdas Vaaran and Gurbani by Sikh Gurus; Guru Nanak Dev, Guru Angad Dev, Guru Amardas, Guru Ramdass, Guru Arjan Dev, Guru Tegh Bahadur, Guru Gobind Singh and Bhagats like Bhagat Kabir Jee, Sheikh Fareed Jee, Bhagat Jaidev, Bhagat Naamdev, Bhagat Ravidaas, Bhagat Ramanand, Bhagat Parmanand, Bhagat Trilochan, Bhagat Surdas.`,
   },
   '/shabad': {
-    title: suffixAppName('Shabad'),
-    createDescription: () =>
-      `Read shabad online right now. ${basicDescription}`,
+    createTitle: (title = 'Shabad') => suffixAppName(title),
+    createDescription: (description = 'Read shabad ') => `${description} - ${BASIC_META_DESCRIPTION}`,
```

Comment:
Don't add BASIC_META_DESCRIPTION here. It might have negative SEO impact (there's a length limit for description) and will create unnecessary noise.